integration/build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
     // Google dependencies
     constraints {
         implementation 'com.google.guava:guava:33.0.0-jre'
-        implementation 'org.json:json:20231013' // security vulnerability
+        implementation 'org.json:json:20240303' // security vulnerability
```

Comment:
It would be interesting to try removing this explicit version dependency and see what version we would then inherit. If it's beyond whatever security vulnerability prompted this override, we could then remove the override.